### DISCUSSION TITLE
Using URDF Transmissions

### DISCUSSION THREAD

**Discussion by willian-henrique** [ICube-Robotics/ethercat_driver_ros2#107] [General]
Hey,

I'm trying to use the default URDF Transmission as cited in: https://wiki.ros.org/urdf/XML/Transmission. However, it seems to me that the plugin of the hardware driver should handle this variable (I'm not sure about that). Could you confirm if the hardware driver's role should handle this variable? In this case, the application in this repository.

Thanks,

Willian Henrique

**Comment by bitmeal** [comment]
Not affiliated with the repository, but have recently been implementing transmissions (at least the `SimpleTransmission`) for another hardware interface: The answer is yes. The hardware interface (driver) has to implement loading and usage of the transmission.
